### ISSUE TITLE
Add-ServiceNowAttachment Error - "Item has already been added. Key in dictionary: 'Content-Type'  Key being added: 'Content-Type'"

### ISSUE THREAD

**Issue by JasonDTX** [Snow-Shell/servicenow-powershell#207]
Hiya, Not sure if Add-ServiceNowAttachment is supposed to do multiple files. 

If it is, I think the Content-Type header might be duplicated in the ForEach loop here:

https://github.com/Snow-Shell/servicenow-powershell/blob/19e38411ee978aff2581eb06f5cdb2827042a289/ServiceNow/Public/Add-ServiceNowAttachment.ps1#L173

I think with multi-file you'll want the type to be multipart or octet-stream.  Haven't tried fixing it yet, currently just working around by sending one file at a time.

# Environment

```
Operating System: Windows Powershell
ServiceNow module version: 3.4.1
PowerShell version:  5.1
```

# Steps to reproduce

Use a splat containing two filepath

```
$addServiceNowAttachmentSplat = @{
    ID          = $Ticket.Number
    Table       = 'incident'
    File        = $LogFilePath, $OutputFilePath
}

Add-ServiceNowAttachment @addServiceNowAttachmentSplat
```

# Expected behavior

Multiple files upload to service now incident

# Actual behavior

First item is uploaded, next files generate error
"Item has already been added. Key in dictionary: 'Content-Type'  Key being added: 'Content-Type'"

# Screenshots

![Web capture_14-10-2022_15813_github com](https://user-images.githubusercontent.com/35275171/195933658-95f91eeb-61f3-4769-98ec-ba07813bbfa7.jpeg)

**Comment by gdbarron** [comment]
Thanks for reporting this @JasonDTX.  I've fixed this along with another issue where content type wasn't being set correctly.  Would you mind testing the fix #208?

**Comment by JasonDTX** [comment]
Looks good, posting in 208, closing 207.
